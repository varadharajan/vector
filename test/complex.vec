complex add(complex a, complex b)
{
    return a + b;
}

int vec_main()
{
    i := #(float64(4.5), float64(2.3)) * #(float64(0.), float64(2.));
    j := add(#(3.,4.),#(5., 4.));

    printf("%0.1f + %0.1fj\n", i.re, i.im);
    printf("%0.1f + %0.1fj\n", j.re, j.im); 
    
    return 0;
}

