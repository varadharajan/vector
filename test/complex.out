-4.6 + 9.0j
8.0 + 8.0j
4.5 + 10.0j
11.3
